src/shared/ui/date-lang/date-lang.tsx

```diff
@@ -1,30 +1,30 @@
-/* eslint-disable @stylistic/jsx-one-expression-per-line */
+import classNames from 'classnames/bind';
 import micIcon from '@/shared/assets/icons/mic.svg';
 import noteIcon from '@/shared/assets/icons/note-icon.svg';
 import Image from '@/shared/ui/image';
 
 import styles from './date-lang.module.scss';
 
+const cx = classNames.bind(styles);
+
 interface DateLangProps {
   startDate: string;
   mode: string;
   language: string[];
-  type?: string;
 }
 
-export const DateLang = ({ startDate, language, mode, type = '' }: DateLangProps) => {
-  const splittedLanguage = language.slice().join('/');
-
+export const DateLang = ({ startDate, language, mode }: DateLangProps) => {
   return (
-    <div className={`${styles.info} ${styles[type]}`}>
-      <p className={styles.date}>
-        <Image className={styles.note} src={noteIcon} alt="note-icon" lazy="false" />
-        <span>Start date {startDate ?? 'Not set'}</span>
+    <div className={cx('info')}>
+      <p className={cx('date')}>
+        <Image className={cx('icon')} src={noteIcon} alt="note-icon" lazy="false" />
+        <span>{`Start date: ${startDate ?? 'not set'}`}</span>
```

Comment:
```suggestion
        <span>{`Start date: ${startDate || 'TBD'}`}</span>
```

Reply:
I'v replaced it with small util function to make consistency with dates in header and course page.  Cuz old code actually doesn't work, there was course dates in data for all courses, so 'TBD' was never set here.